frontend/src/utils/render_utils.jsx

```diff
@@ -12,11 +12,15 @@ export const modelFiles = {
     portal: './basic/portal.glb',
     castle: './basic/house.glb',
     flag: './basic/flag.glb',
-    arrow: './basic/arrow.glb',
-    rock: './basic/cannonball.glb',
     arrowTower: './basic/ballista_tower.glb',
+    arrow: './basic/arrow.glb',
```

Comment:
Prolly worth making this list alphabetical at this point

Reply:
I was thinking we'd use the order from the menu, and group towers with their projectiles. Is alphabetical a general best practice?

Reply:
Yea, for something like this I believe it makes the most sense. This is meant to serve as a lookup table for the models and is used by various components. So someone editing it might be coming from a different context.

I can see how it may be convenient while working on the menu but it kind of abandons the other half of the list. And based on the current naming, the towers will still kept next to their projectiles too.